.github/workflows/cd.yaml

```diff
@@ -52,11 +37,10 @@ jobs:
         with:
           cache-from: type=gha
           cache-to: type=gha,mode=max
-          context: ${{ matrix.context }}
+          context: .
           platforms: linux/amd64,linux/arm64
```

Comment:
I would also love to remove the build step for `linux/arm64` as it seems to make the build CI job take longer than it needs to, but I'll leave it in place for now since I'm not completely sure that it's not used anywhere. Alternatively, I'd like to see if we can parallelize the builds.